configs/vision/pathology/offline/segmentation/total_segmentator_2d.yaml

```diff
@@ -51,17 +52,18 @@ model:
     decoder:
       class_path: eva.vision.models.networks.decoders.segmentation.ConvDecoderMS
       init_args:
-        in_features: ${oc.env:DECODER_IN_FEATURES, 384}
-        num_classes:  &NUM_CLASSES 118
+        in_features: ${oc.env:IN_FEATURES, 384}
+        num_classes:  &NUM_CLASSES 5
```

Comment:
why are config changes part of this pr?

Reply:
For classification this is called `IN_FEATURES`, but there we have a "head" not a "decoder".
I thought it would be easier / less confusing to have the same name for classification & segmentation.

Reply:
ups the number of classes shouldn't change though. good catch

Reply:
fixed now